index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
```

Comment:
- [ ] Unnecessary multiple line spacing may be avoided. While it may be good practice to leave out a space for code readability, it can also be hard to read code that has multiple line spaces for no evident reason